chef/cookbooks/provisioner/libraries/repositories.rb

```diff
@@ -74,12 +74,9 @@ def inspect_repos(node)
       # they can be used.
       def get_repos(platform, version)
         repositories = {}
-        repos_from_databag = Chef::DataBag.load("repositories").keys rescue []
-        repos_from_databag.each do |reponame|
-          dbi = Chef::DataBagItem.load("repositories", reponame)
-          if dbi["platform"] == "#{platform}-#{version}"
-            repositories[dbi["name"]] = { url: dbi["url"] }
-          end
+        repos_db = Chef::DataBag.load("crowbar/repositories") || {}
```

Comment:
https://github.com/crowbar/crowbar-core/pull/102#discussion_r42999096 shows that you have to catch a `Net::HTTPServerException` if the data bag doesn't exist. If I look at the chef code I can not see a case where it returns `nil`.